components/tooltip/tooltip.js

```diff
@@ -931,25 +946,49 @@ class Tooltip extends RtlMixin(LitElement) {
 		this.showing = this._isFocusing || this._isHovering || this.forceShow;
 	}
 
-	_updateTarget() {
+	async _updateTarget() {
 		this._removeListeners();
-		this._target = this._findTarget();
+
+		if (this._target && this.id) {
+			elemIdListRemove(this._target, 'aria-labelledby', this.id);
+			elemIdListRemove(this._target, 'aria-describedby', this.id);
+			if (this.#targetDelegated) {
+				this._target.removeAttribute('aria-label');
+				this._target.removeAttribute('aria-description');
+			}
+		}
```

Comment:
If we already have a target, we're likely updating it, so clean up the existing one.